react-frontend/src/Authentication/Registration.js

```diff
@@ -9,21 +9,35 @@ const Registration = () => {
   const [user, setUser] = useState({
     username: '',
     password: '',
+    confirm_password: '',
   });
   const [register, setRegister] = useState(false);
 
   const handleChange = (event) => {
     const { name, value } = event.target;
-    if (name === 'password') { setUser({ username: user.username, password: value }); } else setUser({ username: value, password: user.password });
+    console.log(name, value);
```

Comment:
Don't forget to remove console.log statements. There is also a security issue associated with this since value could include the password.